### DISCUSSION TITLE
IDE Warning for family parameter even when using freezed

### DISCUSSION THREAD

**Discussion by chanan** [rrousselGit/riverpod#2212] [Q&A] [answered]
Hello,

I am getting a warning from Android Studio when using "family" even when using freezed:

```
final data = ref.watch(
        mediaDownloadProvider(
          MediaDownloadParameters(
            filename: filename!,
            account: account,
          ),
        ),
```

The warning is:

```
Providers parameters should have a consistent ==. Meaning either the values should be cached, or the parameters should override ==
```

MediaDownloadParramers is defined like so:

```
part 'download_media.freezed.dart';

@freezed
class MediaDownloadParameters with _$MediaDownloadParameters {
  const factory MediaDownloadParameters({
    required String filename,
    required Account account,
  }) = _MediaDownloadParameters;
}
```

Not sure if it matters, but I will add just in case it does, Account is also a "freezed" class

Why am I getting that warning if freezed defines the `==` operator?

**Comment by rrousselGit** [comment, marked as answer]
It's a false positive. There's an issue about it. I'll get fixed soon

**Reply by Origogi** [reply]
@rrousselGit 
when did it fixed??
I check new version, but this warning reamained

**Reply by rrousselGit** [reply]
If you have a problem make an issue with a sample to reproduce. It works for me
